### ISSUE TITLE
Fix floating menu focus border (mobile)

### ISSUE THREAD

**Issue by owi92** [elan-ev/tobira#1441]
The focus border is cut off at about 410px screen width while our docs say we (aim to) support screenwidths > 320px. Also the border radius for the top item is wrong. Not sure how relevant focus styles are for mobile devices but this should be fixed regardless.

<img width="506" alt="Image" src="https://github.com/user-attachments/assets/e790c577-e493-49b9-9944-4e1593ea5ae7" />
